tests/system/sseclient/__init__.py

```diff
@@ -0,0 +1,139 @@
+"""
```

Comment:
btw. what is the advantage of pulling in a library code like this?

I mean the code in the original repo is unit tested and we lose any future bugfixes...

Reply:
I want to limit the dependencies as we don't use virtualenv or Pipenv and the like to provision and environment with all dependencies. I tried to use Pipenv but failed. See https://github.com/pypa/pipenv/issues/931 for details.

Reply:
ah, get it... thanks for the explanation!

Reply:
#5831 is a proposal for Pipenv.